yt_dlp/extractor/navernow.py

```diff
@@ -0,0 +1,198 @@
+from .naver import NaverBaseIE
+from ..utils import (
+    ExtractorError,
+    unified_strdate,
+    unified_timestamp,
+    merge_dicts,
+)
+from ..compat import (
+    compat_urllib_parse_urlparse,
+    compat_parse_qs,
+)
+
+
+class NaverNowIE(NaverBaseIE):
+    IE_NAME = 'navernow'
+    _VALID_URL = r'https?://now\.naver\.com/show/(?P<id>[0-9]+)'
+    _PAGE_SIZE = 30
+    _TESTS = [{
+        'url': 'https://now.naver.com/show/4759?shareReplayId=5901#replay=',
+        'info_dict': {
+            'id': '4759-5901',
+            'title': '아이키X노제\r\n💖꽁냥꽁냥💖(1)',
+            'ext': 'mp4',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'timestamp': 1650369600,
+            'upload_date': '20220419',
+            'uploader_id': 'now',
+            'uploader': '',
+            'uploader_url': '',
```

Comment:
Why are these empty strings?

Reply:
These should not be empty strings. I wont ask you to fix the baseIE, but don't add wrong data to tests